scipy/special/_faddeeva.cxx

```diff
@@ -128,6 +128,18 @@ double faddeeva_voigt_profile(double x, double sigma, double gamma)
     const double INV_SQRT_2 = 0.707106781186547524401;
     const double SQRT_2PI = 2.5066282746310002416123552393401042;
 
+    if(sigma == 0){
+        if (gamma == 0){
+            if (x == 0)
+                return NPY_INFINITY;
+            return 0;
+        }
+        return gamma / NPY_PI / (x*x + gamma*gamma);
+    }
+    if (gamma == 0){
+        return 1 / SQRT_2PI / sigma * exp(-x*x / 2 / (sigma*sigma));
```

Comment:
squaring `x/sigma` might be a bit more numerically stable

Reply:
@ilayn, changed it